src/main/java/seedu/address/model/Model.java

```diff
@@ -24,6 +26,15 @@ public interface Model {
      */
     ReadOnlyUserPrefs getUserPrefs();
 
+    Patient getPatient(IcNumber icNumber, List<Patient> patientList);
+    /*public Patient getPatient(IcNumber icNumber, List<Patient> patientList){
+        for (int i = 0; i < patientList.size(); i++) {
+            if(patientList.get(i).getIcNumber().equals(icNumber))
+                return patientList.get(i);
+        }
+        return null;
+    }*/
+
```

Comment:
Would be good if we remove unused code!!

Reply:
Yup noted!